app/models/manageiq/providers/redhat/infra_manager/ovirt_services/strategies/v4.rb

```diff
@@ -815,6 +810,27 @@ def configure_vnic_with_requested_data(name, requested_vnic, destination_vnic, n
       )
     end
 
+    def private_load_allowed_networks(vlans, uid_ems_cluster)
+      profiles = get_vnic_profiles_in_cluster(uid_ems_cluster)
+      profiles.each do |profile, profile_network|
+        vlans[profile.id] = "#{profile.name} (#{profile_network.name})"
+      end
+
+      vlans['<Empty>'] = _('<No Profile>')
+      vlans['<Template>'] = _('<Use template nics>')
+    end
+
+    def parse_vnic_profile_id(requested_profile, uid_ems_cluster)
+      if requested_profile.include?('(')
```

Comment:
Can you please give a name to this? Because I am not sure it will be clear what having '(' means.